lib/declarations.d.ts

```diff
@@ -691,3 +691,18 @@ interface IFrameworkVersion {
 	 */
 	displayName: string;
 }
+
+interface IWebViewService {
+	supportedWebViews: IDictionary<IWebView[]>;
+	getWebViews(platform: string): IWebView[];
+	getWebViewNames(platform: string): string[];
```

Comment:
Yes, we need it. I use it here https://github.com/Icenium/icenium-cli/pull/1061/files#diff-8e82905affabe763582885aec9e511eaR33